### DISCUSSION TITLE
Best settings for accurate live transcription with stream on a Macbook Air M1 (16gb)

### DISCUSSION THREAD

**Discussion by panomitrius** [ggml-org/whisper.cpp#633] [Q&A]
I'm wondering what the best parameters for running stream with either ggml-medium.bin or ggml-large.bin would be - to have a responsive and fairly accurate (non-English) transcription on a Macbook Air M1 (16gb)?
I've been trying around and can't quite get it to work satisfyingly.

**Comment by dev-msp** [comment]
More info would be helpful. What language are you transcribing? What have you tried, and how is it falling short of your expectations?
